readme.md

```diff
@@ -109,6 +109,14 @@ try {
 	}
 	*/
 }
+
+// Killing a spawned process gently, then with more force
```

Comment:
I think the three comments could be summarized in just one, such as:

```suggestion
// If the following spawned process is not terminated by `SIGTERM`, `SIGKILL` will automatically be sent after 2 seconds.
```

Reply:
Fixed: 
* 2fb0ae0a7664c4c240d3ae9f764b6b0976549827
* 233e222679fcbce4b1547370bffcd5bffe32c7c4